LICENSE

```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```

Comment:
Can you change this license to CC BY 4.0 instead?
It's [here](https://choosealicense.com/licenses/cc-by-4.0/)